### PR TITLE
MoviesPage structure + example Tile elements

### PR DESCRIPTION
**Pull request by MikeAdamczyk**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,22 +1,30 @@
 <!DOCTYPE html>
-<html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <!--TODO replace
+<html lang="pl">
```
Comment (anetaszynal) [inline public/index.html:2]: I think that we should use english language according to instruction.

File: src/App.js [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import React from "react";
-import "./App.css";
+import { MoviesPage } from "./features/movies/MoviesPage";
 
-function App() {
-  return <></>;
-}
+export default () => (
```
Comment (anetaszynal) [inline src/App.js:4]: Please keep the previous name for better consistency

File: src/common/components/Paging/styled.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import styled from "styled-components";
+
+export const Footer = styled.footer`
+    text-align: center;
+    justify-content: center;
```
Comment (damdlani) [inline src/common/components/Paging/styled.js:5]: It works for `display: flex` which isn't set here. Btw text-align does the same thing.

File: src/common/components/Title/styled.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+export const TitleBox = styled.div`
+    max-width: 1368px;
+    margin: auto;
+`;
+
+export const MainTitle = styled.h1`
+    color: black;
+    font-weight: 600px;
+    font-size: 36px;
+    margin: 56px 0 24px 0;
```
Comment (damdlani) [inline src/common/components/Title/styled.js:13]: line-height: 120%; is missing :)

File: src/GlobalStyle.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    }
+
+    *, ::after, ::before {
+        box-sizing: inherit;
+    }
+
+    body {
+        font-family: 'Poppins', sans-serif;
+        background-color: ${({ theme }) => theme.color.lightGrey};
+        margin: 0px auto 103px;
```
Comment (anetaszynal) [inline src/GlobalStyle.js:15]: It seems that this margin should be added to the pagination.

**Review by damdlani — COMMENTED** [review]
Great job!

File: src/common/components/Container/index.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    grid-gap: 24px;
+`;
+
+export const TileExample = styled.div`
+    width: 324px;
+    height: 650px;
+    box-shadow: 0px 4px 12px rgba(186, 199, 213, 0.5);
+    background: white;
```
Comment (anetaszynal) [inline src/common/components/Container/index.js:15]: border radius is missing

File: src/common/components/Header/styled.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+
+export const HeaderBox = styled.div`
```
Comment (anetaszynal) [inline src/common/components/Header/styled.js:3]: What do you think to use <header> here?

File: src/common/components/Header/styled.js [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+    background-color: ${({ theme }) => theme.color.black};
+    color: ${({ theme }) => theme.color.white};
+    max-height: 94px;
+ `;
+
+export const Subheader = styled.div`
+    max-width: 1368px;
+    height: 94px;
```
[the remark this answers is not in the document]
Comment (anetaszynal) [reply src/common/components/Header/styled.js:11]: How the subheader is used? What's in it?

File: src/common/components/Search/styled.js [hunk 9]
```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components";
+
+export const Input = styled.input`
+    padding: 10px;
+    background: #FFFFFF;
+    border: 1px solid ${({ theme }) => theme.color.grey};
+    box-sizing: border-box;
```
Comment (anetaszynal) [inline src/common/components/Search/styled.js:7]: In my opinion, it is not necessary here. You declare it earlier

File: src/common/components/Container/index.js [hunk 10]
```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+export const Container = styled.div`
```
Comment (anetaszynal) [inline src/common/components/Container/index.js:3]: What do you think about applying for example <main> ?

File: src/common/components/Container/index.js [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+    grid-gap: 24px;
+`;
+
+export const TileExample = styled.div`
+    width: 324px;
+    height: 650px;
+    box-shadow: 0px 4px 12px rgba(186, 199, 213, 0.5);
+    background: white;
+    color: red;
+    padding: 30px;
```
Comment (anetaszynal) [inline src/common/components/Container/index.js:17]: Use theme, please ;)

Comment (anetaszynal) [inline src/common/components/Container/index.js:17]: I think that you should use 16px padding

File: src/common/components/Menu/Logo/index.js [hunk 12]
```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+
+export const Logo = styled.a`
```
Comment (anetaszynal) [inline src/common/components/Menu/Logo/index.js:3]: width is missed

File: src/common/components/Paging/styled.js [hunk 13]
```diff
@@ -0,0 +1,13 @@
...
+    padding: 11px;
```
Comment (anetaszynal) [inline src/common/components/Paging/styled.js:6]: Do you need it?

File: src/common/components/Paging/styled.js [hunk 14]
```diff
@@ -0,0 +1,13 @@
...
+    margin: 40px auto;
+    font-size: 14px;
+    max-width: 1368px;
```
Comment (anetaszynal) [inline src/common/components/Paging/styled.js:9]: In my opinion, max- width should be 525px

Comment (anetaszynal) [inline src/common/components/Paging/styled.js:9]: Isn't the width constant?

File: src/features/movies/MoviesPage/index.js [hunk 15]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Container, TileExample } from "../../../common/components/Container/index";
+import { Header } from "../../../common/components/Header/index";
```
Comment (anetaszynal) [inline src/features/movies/MoviesPage/index.js:3]: Could you change name for better consistency ?

**Review by anetaszynal — CHANGES_REQUESTED** [review]
It's nice that you did so much. I added a few suggestions in the comments.

File: src/common/components/Header/styled.js [hunk 16]
```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+
+export const HeaderBox = styled.div`
+    background-color: ${({ theme }) => theme.color.black};
+    color: ${({ theme }) => theme.color.white};
+    max-height: 94px;
+ `;
```
[the remark this answers is not in the document]
Comment (MikeAdamczyk) [reply src/common/components/Header/styled.js]: it's 100% by default :)

File: src/common/components/Header/styled.js (hunk 8 above)
[the remark this answers is not in the document]
Comment (MikeAdamczyk) [reply src/common/components/Header/styled.js]: Subheader is needed to set position of menu and search input in proper way. Subheader height: 100% equals 88px, I don't know why :)  so I set min-height: 94px, later on it is changed on the mobile view for 142px.

File: src/common/components/Title/styled.js (hunk 4 above)
Comment (MikeAdamczyk) [reply src/common/components/Title/styled.js:18]: done :)

File: src/common/components/Container/index.js (hunk 6 above)
Comment (MikeAdamczyk) [reply src/common/components/Container/index.js]: it was example Tile, already deleted ;)

File: src/common/components/Header/styled.js (hunk 7 above)
Comment (MikeAdamczyk) [reply src/common/components/Header/styled.js:3]: Name for this section is used according to the project :)

File: src/common/components/Container/index.js (hunk 10 above)
Comment (MikeAdamczyk) [reply src/common/components/Container/index.js:3]: done :) additionally divided for MoviesContainer and PeopleContainer

File: src/common/components/Container/index.js [hunk 17]
```diff
@@ -0,0 +1,19 @@
...
+    font-size: 24px;
```
[the remark this answers is not in the document]
Comment (MikeAdamczyk) [reply src/common/components/Container/index.js]: nope ;) it's just a plug for development :)

File: src/common/components/Menu/Logo/index.js (hunk 12 above)
Comment (MikeAdamczyk) [reply src/common/components/Menu/Logo/index.js]: no need,  and <p> have it's own width and fill it :)

File: src/common/components/Paging/styled.js (hunk 13 above)
Comment (MikeAdamczyk) [reply src/common/components/Paging/styled.js:6]: plug for development ;)

File: src/common/components/Paging/styled.js (hunk 14 above)
Comment (MikeAdamczyk) [reply src/common/components/Paging/styled.js:9]: will be developed later, it's just a plug

File: src/features/movies/MoviesPage/index.js (hunk 15 above)
Comment (MikeAdamczyk) [reply src/features/movies/MoviesPage/index.js]: which name?

File: src/common/components/Header/styled.js (hunk 7 above)
Comment (anetaszynal) [reply src/common/components/Header/styled.js:3]: I meant replacing div with header tag ;)

Comment (MikeAdamczyk) [reply src/common/components/Header/styled.js]: OK, that's good idea :) DONE!

Comment (anetaszynal) [reply src/common/components/Header/styled.js]: Sorry, it's ok ;)

**Review by anetaszynal — APPROVED** [review]
Good job! For my part, everything is fine 😉

**Pull request merged**